lib/node_modules/@stdlib/stats/base/meanpn/test/test.meanpn.js

```diff
@@ -40,6 +41,23 @@ tape( 'the function has an arity of 3', function test( t ) {
 	t.end();
 });
 
+tape( 'the function calculates the arithmetic mean of a strided array (accessors)', function test( t ) {
+	var x;
+	var v;
+
+	x = [ 1.0, -2.0, -4.0, 5.0, 0.0, 3.0 ];
+
+	v = meanpn( x.length, toAccessorArray(x), 1 );
+	t.strictEqual( v, 0.5, 'returns expected value' );
+
+	x = [ -4.0 ];
+
+	v = meanpn( x.length, toAccessorArray(x), 1 );
+	t.strictEqual( v, -4.0, 'returns expected value' );
+
+	t.end();
+});
+
 tape( 'the function calculates the arithmetic mean of a strided array', function test( t ) {
```

Comment:
You need to add explicit test blocks below which test accessor behavior just as we do for indexed arrays. You did one test block, but you are missing all the others for L61 and below.